www/content/docs.md

```diff
@@ -152,36 +152,79 @@ For npm-style build systems, you can install htmx via [npm](https://www.npmjs.co
 npm install htmx.org@2.0.1
 ```
 
-After installing, you’ll need to use appropriate tooling to use `node_modules/htmx.org/dist/htmx.js` (or `.min.js`).
-For example, you might bundle htmx with some extensions and project-specific code.
+After installing, you’ll need to use appropriate tooling to use `node_modules/htmx.org/dist/htmx[.esm, .cjs, .amd].js` (or `.min.js`).
 
-### Webpack
+In the simplest case all you need to do is to add this import to your entry point (like `index.js`)
+```js
+import 'htmx.org';
+```
+
+However, you will probably need the global `htmx` variable, which gives your code access to methods like
+`htmx.onLoad` and allow you to use npm-installed htmx plugins. You can use one of the following solutions
+to achieve this.
 
-If you are using webpack to manage your javascript:
 
-* Install `htmx` via your favourite package manager (like npm or yarn)
-* Add the import to your `index.js`
+#### Global variable 
+
+Create a local `htmx.js` with the following content:
 
 ```js
-import 'htmx.org';
+import htmx from "htmx.org";
+window.htmx = htmx;
 ```
 
-If you want to use the global `htmx` variable (recommended), you need to inject it to the window scope:
+Import this file in your entry point (like `index.js`):
+
+```js
+import './htmx.js';
+
+// now you can import extensions
+import 'htmx-ext-debug/debug.js';
```

Comment:
You are describing a workaround which requires creating `htmx.js` file to inject `htmx` into the `window` scope, so that the extensions can use it. It has to be done in a separate file, because ESM imports are asynchronous.

I have created a [PR](https://github.com/bigskysoftware/htmx-extensions/pull/123) in the extensions repository, which eliminates the need of this workaround. With the new change extensions can be imported like this:
```js
import `htmx.org`;
import `htmx-ext-extension-name`; // Replace `extension-name` with the extension name
```

It relies on the ESM import mechanism supported by most bundlers, including Webpack, Vite and Rollup.